app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AccountActivationsController < ApplicationController 
+  
+  def edit
+    user = User.find_by(email: params[:email])
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
+      user.update_attribute(:activated, true)
+      user.update_attribute(:activated_at, Time.zone.now)
```

Comment:
It's generally preferable to use `update_attributes` over `update_attribute` as well since the former invokes the model validations while the latter skips them.